test/elixir_internal_certificate/account/accounts_test.exs

```diff
@@ -0,0 +1,181 @@
+defmodule ElixirInternalCertificate.Account.AccountsTest do
+  use ElixirInternalCertificate.DataCase
+
+  alias ElixirInternalCertificate.Account.Accounts
+  alias ElixirInternalCertificate.Account.Schemas.{User, UserToken}
+
+  describe "get_user_by_email/1" do
+    test "with the existed email, returns the user" do
+      %{id: id} = insert(:user, email: "random_email@mail.com")
+
+      assert %User{id: ^id} = Accounts.get_user_by_email("random_email@mail.com")
+    end
+
+    test "when the email does not exist, return nil" do
+      assert Accounts.get_user_by_email("unknown@example.com") == nil
+    end
+  end
+
+  describe "get_user_by_email_and_password/2" do
+    test "when the email and password are valid, returns the user" do
+      %{id: id, email: email} = insert(:user)
+
+      assert %User{id: ^id} = Accounts.get_user_by_email_and_password(email, valid_user_password())
+    end
+
+    test "when the email does not exist, return nil" do
+      assert Accounts.get_user_by_email_and_password("unknown@example.com", "hello world!") == nil
+    end
+
+    test "when the password is not valid, does not return the user" do
```

Comment:
How about replacing the `does not return the user` -> returns nil same above? please check for other places as well :)